etc/ReplayOfflineConfiguration.py

```diff
@@ -117,15 +117,15 @@
 hiTestppScenario = "ppEra_Run3"
 
 # Procesing version number replays
-dt = 212
+dt = 410
 defaultProcVersion = dt
 expressProcVersion = dt
 alcarawProcVersion = dt
 
 # Defaults for GlobalTag
-expressGlobalTag = "122X_dataRun3_Express_v3"
-promptrecoGlobalTag = "122X_dataRun3_Prompt_v3"
-alcap0GlobalTag = "122X_dataRun3_Prompt_v3"
+expressGlobalTag = "123X_dataRun3_Express_Candidate_2022_04_08_23_32_25"
+promptrecoGlobalTag = "	123X_dataRun3_Prompt_Candidate_2022_04_08_23_36_44"
+alcap0GlobalTag = "	123X_dataRun3_Prompt_Candidate_2022_04_08_23_36_44"
```

Comment:
@germanfgv I suggest to use the new versioned GTs (that you put in the prod configuration) also here in the replay configuration.